packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -146,6 +147,17 @@ const _FileUpload: React.ForwardRefRenderFunction<BladeElementRef, FileUploadPro
     setIsActive(false);
   };
 
+  const handleFileChangeEvents = ({
+    fileList,
+    name,
+  }: {
+    fileList: BladeFileList;
+    name?: string;
+  }): void => {
+    onChange?.({ name, fileList });
+    onInput?.({ name, fileList });
```

Comment:
https://github.com/razorpay/blade/pull/2401/files#r1830421650

Check this once with Anurag

Reply:
checked closing these PR

Reply:
checked closing these PR